STEERING-COMMITTEE.md

```diff
@@ -162,13 +120,157 @@ There are two ways to raise issues to the steering committee for decision:
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
 
+## Composition
+
+The steering committee will have 7 seats.  Five of the seats will be
+Contributing seats, open to any project contributor.  Two of the seats will be
+End-User seats, open only to active end-user community members.  See Candidate
+Eligibility for definitions of these categories.
+
+Steering Committee members will serve for 2-year terms, staggered in order to
+preserve continuity.  Every year either two or three contributor seats will be
+elected, and one end user seat will be elected.
+
+## Election Procedure
+
+### Timeline
+
+Steering Committee elections shall be held in October or November of each year.
+Six weeks or more before the election, the Steering Committee shall appoint
+Election Officer(s) (see below).  Four weeks or more before the election, the
+Election Officer(s) will issue a call for nominations, publish the list of
+voters, and open the call for exceptions.  One week before the election the call
+for nominations and exceptions will be closed.  The election will be open for
+voting not less than two weeks and not more than four.  The results of the
+election shall be announced within one week of closing the election.  New
+Steering Committee members will take office on December 1 of each year.
+
+### Election Officer(s)
+
+Six weeks or more before the election, the Steering Committee will appoint
+between one and three Election Officer(s) to administer the election.  Elections
+Officers will be Knative community members in good standing who are eligible to
+vote, are not running for Steering in that election, and can make a public
+promise of impartiality.  They will be responsible for:
+
+* Making all announcements associated with the election
+* Preparing and distributing electronic ballots
+* Judging exception requests
```

Comment:
Not sure if we want to send these to steering as a feedback loop, or keep it with the 1-3 election officers.

Reply:
I don't think exception requests will normally be difficult to judge, and I would trust election officers to do so personally. Do you think any text should change here?

Reply:
The feedback loop was to make steering aware of the sorts of work that we weren't currently judging well, but I'm fine either way.

Reply:
FWIW, Kubernetes does it with the decisions made by the EC, with a record kept of each decision.